ptf/tests/config/basic_config.py

```diff
@@ -16,16 +17,80 @@ def config_meta_port(self, test):
         """
         Default configuation, with metadata and ports configurations.
         """
+        self.recreate_ports(test)
         self.get_port_list(test)        
         self.get_default_1q_bridge(test)
         self.get_default_vlan(test)
         self.remove_vlan_member(test)
         self.remove_bridge_port(test)
         self.create_host_intf(test)
+        self.create_bridge_port(test)
         self.turn_on_port_admin_state(test)
         #test.set_port_serdes()
 
 
+    def recreate_ports(self, test):
+        '''
+        Recreate the port base on file specified in 'port_config_ini' param.
+        '''
+        #TODO check if this is common behivor or specified after check on more platform
+        if 'port_config_ini' in test.test_params:
+            if 'createPorts_has_been_called' not in config:
+                self.createPorts(test)
```

Comment:
This method was named as create but not recreate cause we need to compatible with new sai ptf cases. Now we have our own, do we need to?